2.05-custom-gx/defines/mod.hsp

```diff
@@ -6823,6 +6823,7 @@
 #define global	TWEAK_CONVENIENCE_RELOAD_AFTER_DEATH_OPTION	52
 #define global	TWEAK_CONVENIENCE_SKIP_SKILL_TRAINING_CONFIRMATION	53
 #define global	TWEAK_CONVENIENCE_AUTODROP_OPENED_CHESTS	54
+#define global  TWEAK_CONVENIENCE_REMOVE_SPELL_OVER_CAST_CONFIRMATION
```

Comment:
This lacks the constant definition (`55`) so the game crashes when you try setting the tweak.

Reply:
Doh! Thank you for letting me know :>